### PR TITLE
Adding modified Alma fixtures in order to ensure that the test suites for advanced_location_s and location_code_s/location_display are now passing

### LINKED ISSUE #1247: Delete  skipped elf-related tests in traject config spec
https://github.com/pulibrary/bibdata/blob/alma/spec/marc_to_solr/lib/config_spec.rb#L449-L457

https://github.com/pulibrary/bibdata/blob/alma/spec/marc_to_solr/lib/config_spec.rb#L484-L487

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #1247

### CODE REVIEW

**Review by christinach — CHANGES_REQUESTED** [review]
@jrgriffiniii in Alma we don't have elf locations. We should not test with modified elf fixtures.

**Comment by christinach** [comment]
In other words we should remove elf tests. Because of the initial migration elf location still exists in alma but it is a cleanup issue and it will be removed. Also when we generate the locations tables we exclude the elf locations.

**Comment by christinach** [comment]
We are testing online locations using the electronic_portfolio_s field, so we can close this pr. I'm sorry that the issue was not updated after we added the electronic_portfolio_s field.

**Comment by christinach** [comment]
for reference I'm adding my comment from the issue https://github.com/pulibrary/bibdata/issues/1247#issuecomment-861744423. it references an example and the test for the electronic portfolio_s field.

**Comment by jrgriffiniii** [comment]
I am very sorry for misunderstanding the scope of this issue, I will proceed with simply deleting the tests.

File: spec/fixtures/marc_to_solr/9992320213506421_elf1.mrx [hunk 1]
```diff
@@ -0,0 +1,147 @@
+<record>
+  <leader>01102cam a2200325 a 4500</leader>
+  <controlfield tag="005">20201006134116.0</controlfield>
```
Comment (christinach) [inline spec/fixtures/marc_to_solr/9992320213506421_elf1.mrx:3]: @jrgriffiniii can you please remove this fixture?

File: spec/fixtures/marc_to_solr/9992320213506421_elf2.mrx [hunk 2]
```diff
@@ -0,0 +1,147 @@
+<record>
+  <leader>01102cam a2200325 a 4500</leader>
+  <controlfield tag="005">20201006134116.0</controlfield>
```
Comment (christinach) [inline spec/fixtures/marc_to_solr/9992320213506421_elf2.mrx:3]: can you please remove this fixture as well?

File: spec/marc_to_solr/lib/config_spec.rb [hunk 3]
```diff
@@ -442,19 +443,26 @@ def fixture_record(fixture_name)
...
-    # TODO: Replace with Alma
-    # Question: Is this still valid?
-    # Revisit while working on https://github.com/pulibrary/marc_liberation/issues/921
-    xit 'when location is online' do
-      expect(@online['location_code_s']).to include 'online$elf1'
-      expect(@online['location_display']).to include 'Electronic Access - elf1 Internet Resources'
-      expect(@online['location']).to eq ['Electronic Access']
+    let(:location_code_s) do
+      current_record['location_code_s']
     end
```
Comment (christinach) [inline spec/marc_to_solr/lib/config_spec.rb:448]: can we make this let one line?

File: spec/marc_to_solr/lib/config_spec.rb [hunk 4]
```diff
@@ -442,19 +443,26 @@ def fixture_record(fixture_name)
...
+    end
+    let(:location) do
+      current_record['location']
+    end
+
+    context 'when there are location codes which do not map to the labels' do
+      let(:id) do
+        '99276293506421_invalid_location'
+      end
+
```
Comment (christinach) [inline spec/marc_to_solr/lib/config_spec.rb:460]: can we also change these lets to one line?

File: spec/marc_to_solr/lib/config_spec.rb [hunk 5]
```diff
@@ -468,22 +476,35 @@ def fixture_record(fixture_name)
       expect(@online_at_library['location']).not_to include 'Mudd Manuscript Library'
     end
   end
+
+  let(:record_fixture_path) do
+    fixture_record(id)
+  end
+  let(:current_record) do
+    @indexer.map_record(record_fixture_path)
+  end
```
Comment (christinach) [inline spec/marc_to_solr/lib/config_spec.rb:485]: can we change these lets to one line?

File: spec/marc_to_solr/lib/config_spec.rb [hunk 6]
```diff
@@ -468,22 +476,35 @@ def fixture_record(fixture_name)
...
-    it 'lewis library included with lewis code' do
-      record = @indexer.map_record(fixture_record('9992320213506421'))
-      expect(record['advanced_location_s']).to include 'lewis$stacks'
-      expect(record['advanced_location_s']).to include 'Lewis Library'
+    let(:id) do
+      '9992320213506421'
+    end
+    let(:location_code_s) do
+      current_record['location_code_s']
     end
```
Comment (christinach) [inline spec/marc_to_solr/lib/config_spec.rb:493]: can we change the let to one line?

**Review by christinach — CHANGES_REQUESTED** [review]

**Review by christinach — APPROVED** [review]

**Pull request merged**